Home.md

```diff
@@ -17,3 +17,73 @@ Use the [`#WindowsCommunityToolkit`](https://twitter.com/search?q=%23WindowsComm
 - Windows Blogs - Release Overviews - [Search for Windows Community Toolkit](https://blogs.windows.com/?s=Windows+Community+Toolkit)
 - Dev Blogs - [Other Articles](https://devblogs.microsoft.com/pax-windows/)
 - We may sometimes add to the [.NET Foundation Newsletter](https://github.com/dotnet-foundation/website/blob/master/input/blog/posts/_current-newsletter-draft.md) too!
+
+## Projects that are currently using Windows Community Toolkit...
+
+<h1 align="center">Fiona</h1>
+
+<table>
+  <tr>
+    <td><img src=https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona1.png height=350></td>
```

Comment:
I am not sure these links are a good idea, we should use images from the `WindowsCommunityToolkit`'s repo. Maybe we can use a relative path to reference them.

Reply:
Let's also have 1 image per app, think the first one for Fiona is the best representation.